user-docs/modules/ROOT/pages/fedora-iot-bootc-pmachine-example.adoc

```diff
@@ -0,0 +1,58 @@
+= Fedora IoT Bootc Image Example with Podman Machine
+
+This example walks through building and booting a Fedora IoT bootc image in a Podman machine. 
+
+To start, create a clone of the https://gitlab.com/fedora/bootc/base-images[Fedora Bootc Base Images] Git repository. This includes the Fedora
```

Comment:
Checking out the git repo shouldn't be necessary; the manifests should be embedded in the `builder` container image below.  (Which needs to be changed...)